aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -384,6 +389,30 @@ Tensor& round_sparse_csr_(Tensor& self) {
   return self;
 }
 
+Tensor threshold_backward_sparse_compressed(
+    const Tensor& grad_output,
+    const Tensor& self,
+    const Scalar& threshold) {
+  return get_result_tensor_for_unary_op(
+      [&](const Tensor& t) {
+        return at::threshold_backward(t, self.values(), threshold);
+      },
```

Comment:
Hmm, is there a guard that makes sure that `self` is coalesced? Otherwise `self.values()` will error out.